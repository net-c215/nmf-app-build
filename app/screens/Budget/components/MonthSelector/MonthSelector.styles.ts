import { StyleSheet } from "react-native";
import { Layout } from "../../../../style";

export default StyleSheet.create({
  container: {
    ...Layout.containerWithPadding,
    paddingVertical: 20,
    flex: 1,
    flexDirection: "row"
  },
  arrowForward: {
    justifyContent: "center",
    flex: 1
  },
  arrowBack: {
    justifyContent: "center",
    alignItems: "flex-end",
    flex: 1
  },
  monthView: {
    justifyContent: "center",
    alignItems: "center",
    flex: 4
  }
});
