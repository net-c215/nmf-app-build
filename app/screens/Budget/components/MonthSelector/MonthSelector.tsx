import React from "react";
import { View, TouchableOpacity } from "react-native";
import styles from "./MonthSelector.styles";
import { Text } from "../../../../components";
import { Ionicons } from "@expo/vector-icons";
import colors from "../../../../style/colors";

const MonthSelector = () => {
  return (
    <View style={styles.container}>
      <TouchableOpacity
        onPress={() => {
          // do nothing.
        }}
        style={styles.arrowForward}
      >
        <Ionicons name={"ios-arrow-back"} size={30} color={colors.swordGray} />
      </TouchableOpacity>
      <View style={styles.monthView}>
        <Text.H3>Month</Text.H3>
      </View>
      <TouchableOpacity
        onPress={() => {
          // do nothing.
        }}
        style={styles.arrowBack}
      >
        <Ionicons
          name={"ios-arrow-forward"}
          size={30}
          color={colors.swordGray}
        />
      </TouchableOpacity>
    </View>
  );
};

export default MonthSelector;
