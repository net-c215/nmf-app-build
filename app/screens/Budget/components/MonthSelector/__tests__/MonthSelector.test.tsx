import React from "react";
import renderer from "react-test-renderer";
import MonthSelector from "../MonthSelector";

jest.unmock("../MonthSelector");

it("MonthSelector renders correctly", () => {
  const tree = renderer.create(<MonthSelector />).toJSON();
  expect(tree).toMatchSnapshot();
});
