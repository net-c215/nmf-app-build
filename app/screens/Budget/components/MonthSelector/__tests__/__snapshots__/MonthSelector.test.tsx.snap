// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthSelector renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "paddingHorizontal": 14,
      "paddingVertical": 20,
    }
  }
>
  <TouchableOpacity
    onPress={[Function]}
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text />
  </TouchableOpacity>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 4,
        "justifyContent": "center",
      }
    }
  >
    <Text.H3>
      Month
    </Text.H3>
  </View>
  <TouchableOpacity
    onPress={[Function]}
    style={
      Object {
        "alignItems": "flex-end",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text />
  </TouchableOpacity>
</View>
`;
